### PR TITLE
Adding my image and some other spice to the mix

### PR DESCRIPTION
**Pull request by postphotos**
According to the [Internet](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a),
> The HTML <a> element (or anchor element) creates a hyperlink to other web pages, files, locations within the same page, email addresses, or any other URL.
Huh, who would've thought... An `index` page might have links to other things, including itself!~ :^)

Some call this to be the base primitive to how the web works, so let me indulge you with some selections for ourselves. We didn't use them for our images, so I made some lovely suggestions. 

I tidied up some of the meh HTML FE code too.  

I need at least two reviewers to feel sane about this.

p.s. Here, we're not really even HTTPSters, just HTTPters. 😢

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,31 +2,44 @@
...
+        <li><a href="https://github.com/postphotos" title="@postphotos">Leo Postovoit</a></li>
     </ul>
-    <hr>
+    <hr id="horizontal-rule">
 
-    <img src="img/arfie.gif" alt="Arfie" title='Arfie says: "Comments and Suggestions Are Welcome"' >
-    <img src="img/netscape.gif" alt="Best viewed with Netscape Navigator" >
-    <img src="img/vim.gif" alt="Created with vim with &heart;" >
-    <img src="img/made-with-emacs.png" alt="Made with Emacs" >
+    <a href="http://www.vintagekennelclub.com/"><img src="img/arfie.gif" alt="Arfie" title='Arfie says: "Comments and Suggestions Are Welcome"' ></a>
```
Comment (MasterKale) [inline index.html]: This needs to link to dogpile.com, the search engine to which Arfie is the official mascot 🐶 

https://web.archive.org/web/19970710110326im_/http://www.dogpile.com:80/

**Review by MasterKale — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (postphotos) [reply index.html]: I think that is fantastic 💟 🐶

Comment (postphotos) [reply index.html]: Let's be real though - vintagekennelclub.com is quite funny too

**Review by MasterKale — APPROVED** [review]

**Review by davidnuon — APPROVED** [review]
lgtm

**Pull request merged**